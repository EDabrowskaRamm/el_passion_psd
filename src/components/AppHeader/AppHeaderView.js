import React from 'react'
import PropTypes from 'prop-types'

import { Root } from '~/components/AppHeader/AppHeader.shards'
import Link from '~/components/Link'
import ImageTag from '~/components/ImageTag'

const AppHeaderView = ({ layoutType }) => (
  <Root className={layoutType}>
    <Link href='/'>
      <ImageTag src='logo.svg' width='300' />
    </Link>
  </Root>
)

AppHeaderView.propTypes = {
  layoutType: PropTypes.string
}

export default AppHeaderView
