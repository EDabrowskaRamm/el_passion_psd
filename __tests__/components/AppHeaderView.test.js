/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'

import AppHeaderView from '~/components/AppHeader/AppHeaderView'

describe('AppHeaderView', () => {
  it('Snapshot', () => {
    const component = renderer.create(<AppHeaderView />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
