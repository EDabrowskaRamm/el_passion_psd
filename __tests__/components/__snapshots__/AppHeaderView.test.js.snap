// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeaderView Snapshot 1`] = `
.emotion-0 {
  grid-area: header;
  padding: 20px;
  border-bottom: 1px solid #e3e3e3;
}

<header
  className="emotion-0 emotion-1"
>
  <a
    href="/"
    onClick={[Function]}
  >
    <img
      src={
        Object {
          "process": [Function],
        }
      }
      srcSet=""
      width="300"
    />
  </a>
</header>
`;
